buildSrc/src/main/java/com/uber/okbuck/core/task/OkBuckCleanTask.java

```diff
@@ -39,11 +40,12 @@ void clean() throws IOException {
     // Get last project paths
     Set<String> lastProjectPaths;
     if (okbuckState.exists()) {
-      lastProjectPaths =
-          Files.lines(okbuckState.toPath())
-              .map(String::trim)
-              .filter(s -> s.length() > 0)
-              .collect(MoreCollectors.toImmutableSet());
+      try (Stream<String> lines = Files.lines(okbuckState.toPath())) {
+        lastProjectPaths =
+            lines.map(String::trim)
+                .filter(s -> s.length() > 0)
```

Comment:
`!Strings.isEmpty()`, or even `!Strings.isBlank()` to rule out whitespace-only text

Reply:
Sure